src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -30,13 +37,27 @@ export function flatten(data) {
 
 export function transform(formConfig, form) {
   const formData = flatten(transformForSubmit(formConfig, form));
+  delete formData.storePrefillStatus;
   return JSON.stringify({
     educationBenefitsClaim: {
       form: formData
     }
   });
 }
 
+export function prefillTransformer(pages, formData, metadata, state) {
+  const newData = cloneDeep(formData);
```

Comment:
You shouldn't need to deep clone this. I think you can use `set` from data-utils on line 52 and set newData to the result. It'll do a shallow clone for you.

Reply:
Great idea, thanks @jbalboni, changed.